sdk/documentintelligence/Azure.AI.DocumentIntelligence/CHANGELOG.md

```diff
@@ -14,3 +14,146 @@
 
 ### Features Added
 - This is the first preview of the `Azure.AI.DocumentIntelligence` package, targeting version 2023-10-31-preview of the Document Intelligence service.
+
+**Note: Document Intelligence is a rebranding of Form Recognizer. While this package and `Azure.AI.FormRecognizer` target the same Azure service, `Azure.AI.DocumentIntelligence` is not backward compatible and should be seen as a new library. See the [Migration Guide][migration_guide] for detailed instructions on how to update application code from `Azure.AI.FormRecognizer` version `4.X.X`. The following list of changes uses version 4.1.0 for comparison to assist during migration.**
```

Comment:
Is this the 2nd preview of the SDK?  I do not see the version update?

When I looked at the below list added property, removed field etc. I assume this is between 2 versions. Let me know if I am misinterpreting the changes.

Reply:
Discussed offline with @kinelski, this is intentional